src/views/sent_sms_list/Pagination.js

```diff
@@ -26,7 +26,7 @@ const Pagination = ({ pager }) => {
     return (
         <div className={s.container}>
             <div className={s.left}>
-                <div>{i18n.t('SMSes per page')}</div>
+                <span>{i18n.t('Show')}</span>
```

Comment:
I remember there was a discussion about this, I didn't pay too much attention, but thought the outcome was to change `[text...] DROPDOWN [text...]` to `[text...] DROPDOWN`.

I think having it like the way you changed it to will make it very difficult for translators as they might only see the individual strings and don't know that these are connected.

Reply:
I agree, I made the same remark. Reason I changed it here is to align it with Hendrik's code for the received sms page, which is closer to Joe's spec. I did that so that they're both aligned. Hendrik I believe is working on the pagination component, and once that's done we can move both of them over to the improved version.